src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -339,13 +339,22 @@ public static partial class RoslynInsertionTool
 
                     try
                     {
-                        await QueueBuildPolicy(pullRequest, "VAL build with RPS");
+                        await QueueBuildPolicy(pullRequest, "[Deprecated] ValBuild RPS");
                     }
                     catch (Exception ex)
                     {
-                        Log.Error($"Unable to create a validation build for '{pullRequest.SourceRefName}'");
+                        Log.Error($"Unable to create a deprecated validation build for '{pullRequest.SourceRefName}'");
+                        Log.Error(ex);
+                    }
+
+                    try
+                    {
+                        await QueueBuildPolicy(pullRequest, "CloudBuild - RPS");
```

Comment:
Not sure what's right here.  There's "CloudBuild - Request RPS" and also "CloudBuild RPS Status" and I'm not sure if the "Status" suffix is included or which one is right?

Reply:
I don't think 'Status' is part of the title because it's listed on PRs as 'CloudBuild - Request RPS succeeded' (or failed).

Reply:
@brettfo So then it should be "CloudBuild - Request RPS" (instead of what I have without the "Request")?

![image](https://user-images.githubusercontent.com/235241/45511677-7e5c9c80-b752-11e8-825a-ea9b5057e17f.png)